src/Util/Images.php

```diff
@@ -347,8 +347,17 @@ public static function getBBCodeByResource(string $resource_id, string $nickname
 	public static function getBBCodeByUrl(string $photo, string $preview = null, string $description = null): string
 	{
 		if (!empty($preview)) {
-			return '[url=' . $photo . '][img=' . $preview . ']' . $description . '[/img][/url]';
+			if (!is_null($description)) {
+				return '[url=' . $photo . '][img=' . $preview . ']' . $description . '[/img][/url]';
+			} else {
+				return '[url=' . $photo . '][img]' . $preview . '[/img][/url]';
```

Comment:
We've had a discussion about how the default generated BBCode for inserted images was cumbersome to edit to allow for image descriptions, please revert to the previous version where `$description` is never expected to be null and we don't use the `[img]Image URL[/img]` form anymore, only the [img=Image URL][/img]` form.

Reply:
See https://github.com/friendica/friendica/pull/13107/files#diff-d87c4b23232100f153fcbb4d5f327ba0882b94ca767226103ca2fb44c5c37385R182 - at upload (where it matters), the format with the easy way to enter a description will be chosen.

Reply:
Yes, but we never need the shorter format, which would simplify the method you created. We never need an actual distinction between not having a description and having an empty description.